### PR TITLE
media-gfx/{apngdis,farbfeld,icon-slicer}: EAPI8 bump

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

another few simple `EAPI8` bump. Also removed the live ebuild from `media-gfx/farbfeld`

### CODE REVIEW

**Comment by orbea** [comment]
> Also removed the live ebuild from `media-gfx/farbfeld`

Just a suggestion, but it would be nice to document why in the commit message.

**Comment by mm1ke** [comment]
> > Also removed the live ebuild from `media-gfx/farbfeld`
> 
> Just a suggestion, but it would be nice to document why in the commit message.

Sure, i've updated the commit. Basically the reason is because the last upstream commit was from 2018 and there is only one more commit from the last version which simple updates the copyright header.

File: media-gfx/icon-slicer/icon-slicer-0.3.ebuild [hunk 1]
```diff
@@ -8,7 +8,7 @@ HOMEPAGE="https://www.freedesktop.org/software/icon-slicer/"
 SRC_URI="https://www.freedesktop.org/software/icon-slicer/releases/${P}.tar.gz"
 
 KEYWORDS="~alpha amd64 ~hppa ~ia64 ppc sparc x86"
-LICENSE="GPL-2"
+LICENSE="MIT"
 SLOT="0"
 IUSE=""
```
Comment (ConiKost) [inline media-gfx/icon-slicer/icon-slicer-0.3.ebuild:13]: If empty, drop it :-)

Comment (mm1ke) [reply media-gfx/icon-slicer/icon-slicer-0.3.ebuild]: Thanks, missed that and fixed now

**Review by ConiKost — APPROVED** [review]

**Pull request closed without merging**